doc/sphinx/conf.py

```diff
@@ -188,13 +188,66 @@ def setup(app):
     # but is it the right place?
     'module',
     'redexpr',
+    'red_expr',
     'modpath',
     'dirpath',
     'collection',
     'term_pattern',
     'term_pattern_string',
     'command',
-    'symbol' ]]
+    'symbol',
+
+    # added for Ltac chapter
```

Comment:
Shouldn't the addition of this list be postponed to the corresponding PR?

Reply:
I can do that.  FWIW, it ended up this way because I had worked on both Ltac and Gallina chunks and having separate changes to orderedGrammar for the 2 parts would be very hard to merge if each started with a semi-random orderedGrammar.  (See my earlier comment on the randomness of orderedGrammar.)  And then the conf.py changes came along for the ride.

Reply:
I get it. It's fine to leave like this if that's less work.